docs/preview/03-Features/powershell/azure-api-management.md

```diff
@@ -192,6 +192,201 @@ PS> Create-AzApiManagementUserAccount `
 # Account has been created for $FirstName $LastName ($mailAddress) for Azure API Management service '$ServiceName' in resource group '$ResourceGroupName'
 ```
 
+## Applying user configuration to an Azure API Management service from a configuration file
+
+Apply user configuration to an existing Azure API Management instance. You can create or update users and assign groups and/or subscriptions to these users.
+
+| Parameter                         | Mandatory | Description                                                                                                                                                         |
+| --------------------------------- | --------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `ResourceGroupName`               | yes       | The resource group containing the Azure API Management instance                                                                                                     |
+| `ServiceName`                     | yes       | The name of the Azure API Management instance located in Azure                                                                                                      |
+| `ConfigurationFile`               | yes       | Path to the JSON Configuration file containing the user configuration                                                                                               |
+| `StrictlyFollowConfigurationFile` | no        | Indicates whether the configuration file should strictly be followed, for example remove the user from groups not defined in the configuration file                 |
+| `ApiVersion`                      | no        | The version of the management API to be used. (default: `2021-08-01`)                                                                                               |
+| `SubscriptionId`                  | no        | The Id of the subscription containing the Azure API Management instance. When not provided, it will be retrieved from the current context (Get-AzContext).          |
+| `AccessToken`                     | no        | The access token to be used to add the user to the Azure API Management instance. When not provided, it will be retrieved from the current context (Get-AzContext). |
+
+**Configuration File**
+
+The configuration file is a simple JSON file that contains the users that need to be created or updated, the JSON file consists of an array of JSON objects.
+
+ The file needs to adhere to the following JSON schema:
+
+``` json
```

Comment:
Can we add a `summary` tag to this so that it is 'hidden' by default and users can flip it open? Think we already have done something like that in the past.

Reply:
Good idea, added in [`7780677` (#422)](https://github.com/arcus-azure/arcus.scripting/pull/422/commits/7780677944b2ce2c27779a30e89c07c1ae9ade90)